src/devtools/client/debugger/src/actions/quick-open.ts

```diff
@@ -26,7 +24,9 @@ export { setQuickOpenQuery, closeQuickOpen };
 
 export function openQuickOpen(query = "", project = false): UIThunkAction {
   return dispatch => {
-    dispatch(loadGlobalFunctions());
+    if (project) {
```

Comment:
Can we also change this variable name to `functionSearch` or something else that reflects why we would pass this flag?

Reply:
Nevermind! This is differentiating between global function search and project function search, so the name kind of makes sense 🤷